test/fixtures/dialects/postgres/select_offset.sql

```diff
@@ -0,0 +1,5 @@
+SELECT * FROM test OFFSET 10;
+
+SELECT * FROM test LIMIT 20 OFFSET 10;
+
+SELECT 1 FROM course_permissions AS cp WHERE cp.course_id = 4 OFFSET 1;
```

Comment:
All the .sql files you're generating across your PRs are missing a final newline. We try not to merge PRs with this missing generally. If you're using VSCode as an IDE, you might want to install the `Editorconfig for VSCode` extension - it'll automatically do a biunch of stuff for your files, such as: standardising tabs and spaces, adding a final newline to files, and deleting trailing whitespace